partner_invoicing_mode/__manifest__.py

```diff
@@ -8,9 +8,11 @@
     "website": "https://github.com/OCA/account-invoicing",
     "license": "AGPL-3",
     "category": "Accounting & Finance",
-    "depends": ["account", "queue_job", "sale"],
+    "depends": ["account", "base_partition", "queue_job", "sale"],
```

Comment:
So the main big change is using a new dependency. We are not using in this moment this module. I didn't know about it

https://github.com/OCA/server-tools/tree/16.0/base_partition

Reply:
It's not a big one. Technical one.

Reply:
I agree, code reuse is always good.